test/unitTestsMovies.js

```diff
@@ -0,0 +1,78 @@
+const test = require("tape");
```

Comment:
If you have a controller.js file, the test file tends to be controller.test.js

Reply:
Now the test works, but i need learn how move to controller.test.js